### DISCUSSION TITLE
XSD's compatible

### DISCUSSION THREAD

**Discussion by ashoktronix** [ZUGFeRD/mustangproject#332] [Q&A] [answered]
Hi Community,

As we know the xsd's for 2.1 and 2.2 are same with minor changes.
Can we validate 2.1.1 version of xml file against the 2.2 XSDs ? it is backward compatible ?

**Comment by jstaerk** [comment, marked as answer]
You mean the XSDs for ZUGFeRD?
>Can we validate 2.1.1 version of xml file against the 2.2 XSDs ? it is backward compatible ?
You can do that, actually you may have to do it as there might be no indication which ZF Subversion you are dealing with. In the worst case you get a valid result for something which was invalid at the time and only later become valid.
Same applies for schematron. And please validate against both schema and schematron.
